fe/src/components/Signup/FormUserDetails.jsx

```diff
@@ -60,20 +87,21 @@ const FormUserDetails = props => {
                     type='submit'
                     onSubmit={props.onSubmit}
                     margin='normal'
-                    halfWidth
                     variant='contained'
                     label='Continue'
                     style={styles.button}
                     onClick={next}
                 >Continue
                 </Button>
                 </Container>
+                </div>
```

Comment:
Please clean-up the indention when you get a chance.

Reply:
Okay I dont know why it got so off but I have Prettier on now so it should clean that up but I'll make sure to double check